src/main/resources/templates/pps/details.html

```diff
@@ -6,51 +6,69 @@
 
 <head>
     <title>
-        Tell us your penalty details
+        Penalty Details
     </title>
 </head>
 
     <div id="lfp-enter-details"  layout:fragment="content">
         <form id="lfpenterdetails" th:action="@{''}" th:object="${enterLFPDetails}" method="post" class="form">
+
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-two-thirds">
 
                     <div th:replace="fragments/lfpGlobalErrors :: globalErrors"></div>
```

Comment:
I think this should be changed to use the new "There is a problem" alternative introduced recently by @klai-ch 
See `templates/fragments/ppsGlobalErrors.html`?

Reply:
Missed that, will updated.